rpcserver.go

```diff
@@ -1779,17 +1779,22 @@ func (r *rpcServer) fetchActiveChannel(chanPoint wire.OutPoint) (
 func (r *rpcServer) GetInfo(ctx context.Context,
 	in *lnrpc.GetInfoRequest) (*lnrpc.GetInfoResponse, error) {
 
-	var activeChannels uint32
 	serverPeers := r.server.Peers()
-	for _, serverPeer := range serverPeers {
-		activeChannels += uint32(len(serverPeer.ChannelSnapshots()))
-	}
 
-	openChannels, err := r.server.chanDB.FetchAllOpenChannels()
+	channels, err := r.server.chanDB.FetchAllOpenChannels()
 	if err != nil {
 		return nil, err
 	}
-	inactiveChannels := uint32(len(openChannels)) - activeChannels
+
+	var nActiveChannels uint32
```

Comment:
No need to prefix any of the variable names. We don't use any sort of prefix/suffix naming conventions within the codebase.